uber/templates/regform.html

```diff
@@ -92,7 +92,6 @@
     $(function () {
         staffingClicked();
         if ($.field('staffing')) {
-            {% if attendee.shifts %} $.field('staffing').prop('readonly', true).prop('title', 'Please {{ "contact your department head to drop shifts" if c.AT_THE_CON else "unassign yourself from shifts" }} before changing your volunteer status.'); {% endif %}
```

Comment:
This is fine, but do you think it's better to just put this line behind a `{% if not admin_area %}` check? That way the checkbox behaves as we want it to on the public pages.